test/ctia/entity/casebook_test.clj

```diff
@@ -353,3 +354,15 @@
                           new-casebook-minimal
                           true
                           true))))
+
+(deftest test-casebook-metric-routes
+  ((:es-store store-fixtures)
+   (fn []
+     (helpers/set-capabilities! "foouser" ["foogroup"] "user" all-capabilities)
+     (whoami-helpers/set-whoami-response "45c1f5e3f05d0" "foouser" "Administrators" "user")
+     (test-metric-routes {:entity :casebook
```

Comment:
the plural namings are already defined on the entity namespace, it would be better to reuse it to avoid duplication

https://github.com/threatgrid/ctia/blob/master/src/ctia/entity/actor.clj#L129